airbyte-webapp/src/views/layout/SideBar/SideBar.tsx

```diff
@@ -36,36 +36,99 @@ export const useCalculateSidebarStyles = () => {
   return ({ isActive }: { isActive: boolean }) => menuItemStyle(isActive);
 };
 
-const SideBar: React.FC = () => {
+interface SideBarProps {
+  additionalTopItems?: JSX.Element;
+  bottomMenuItems?: JSX.Element[];
+}
+
+export const SideBar: React.FC<SideBarProps> = ({ additionalTopItems, bottomMenuItems }) => {
   const config = useConfig();
   const navLinkClassName = useCalculateSidebarStyles();
   const { formatMessage } = useIntl();
 
+  const OSSBottomMenuItems = [
+    <a href={links.updateLink} target="_blank" rel="noreferrer" className={styles.menuItem} data-testid="updateLink">
+      <FontAwesomeIcon className={styles.helpIcon} icon={faRocket} />
+      <Text className={styles.text} size="sm">
+        <FormattedMessage id="sidebar.update" />
+      </Text>
+    </a>,
+    <DropdownMenu
+      placement="right"
+      displacement={10}
+      options={[
+        {
+          as: "a",
+          href: links.docsLink,
+          icon: <DocsIcon />,
+          displayName: formatMessage({ id: "sidebar.documentation" }),
+        },
+        {
+          as: "a",
+          href: links.slackLink,
+          icon: <FontAwesomeIcon icon={faSlack} />,
+          displayName: formatMessage({ id: "sidebar.joinSlack" }),
+        },
+        {
+          as: "a",
+          href: links.tutorialLink,
+          icon: <RecipesIcon />,
+          displayName: formatMessage({ id: "sidebar.recipes" }),
+        },
+      ]}
+    >
+      {({ open }) => (
+        <button className={classNames(styles.dropdownMenuButton, { [styles.open]: open })}>
+          <DocsIcon />
+          <Text className={styles.text} size="sm">
+            <FormattedMessage id="sidebar.resources" />
+          </Text>
+        </button>
+      )}
+    </DropdownMenu>,
+    <NavLink className={navLinkClassName} to={RoutePaths.Settings}>
+      <React.Suspense fallback={null}>
+        <NotificationIndicator />
+      </React.Suspense>
+      <SettingsIcon />
+      <Text className={styles.text} size="sm">
+        <FormattedMessage id="sidebar.settings" />
+      </Text>
+    </NavLink>,
+  ];
```

Comment:
I debated wether to create an `OSSSideBar` component that passes these in (as a parallel to `CloudSideBar`) to keep the main component clean... on the fence as to whether that's warranted.  

It seems ok to me to treat OSS as the base case like this but I could be convinced otherwise.

Reply:
That makes sense!